### PR TITLE
style(example): optimize code styles (F-J)

### PR DESCRIPTION
**Pull request by RSS1102**
## Checklist

List of tasks you have already done and plan to do.

- [x] Fix linting errors
- [x] Tests have been added / updated (or snapshots)

## Change information

Describe your modifications here.

## Issues

no

## Related Links

no

### CODE REVIEW

File: packages/varlet-ui/src/form/example/index.vue [hunk 1]
```diff
@@ -1,22 +1,6 @@
...
-import VarRadioGroup from '../../radio-group'
-import VarRadio from '../../radio'
-import VarButton from '../../button'
-import VarSwitch from '../../switch'
-import VarSlider from '../../slider'
-import VarUploader from '../../uploader'
-import VarCounter from '../../counter'
-import VarRate from '../../rate'
-import VarSpace from '../../space'
-import VarCustomFormComponent from './CustomFormComponent'
```
Comment (haoziqaq) [inline packages/varlet-ui/src/form/example/index.vue:17]: Can't delete here

File: packages/varlet-ui/src/icon/example/index.vue [hunk 2]
```diff
@@ -1,11 +1,5 @@
 <script setup>
-import VarIcon from '..'
-import VarInput from '../../input'
-import vRipple from '../../ripple'
-import Snackbar from '../../snackbar'
-import Clipboard from 'clipboard'
-import icons from '@varlet/icons'
```
Comment (haoziqaq) [inline packages/varlet-ui/src/icon/example/index.vue:7]: please verify module name

File: packages/varlet-ui/src/form/example/index.vue (hunk 1 above)
Comment (RSS1102) [reply packages/varlet-ui/src/form/example/index.vue:17]: `VarCustomFormComponent` not used, 
image
other loaded normally.
image

File: packages/varlet-ui/src/icon/example/index.vue (hunk 2 above)
Comment (RSS1102) [reply packages/varlet-ui/src/icon/example/index.vue:7]:
```ts
import Clipboard from 'clipboard'
import icons from '@varlet/icons'
```
They seem to have been wrongly cited

**Pull request merged**